### PR TITLE
feat: map

### PR DESCRIPTION
**Pull request by EinSev**
added feature: activities map

redesigned entire journey view and correction of a few things

### CODE REVIEW

File: src/components/JourneyMap.vue [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+      .select('pk_activity_uuid, name, address, google_maps_link, added_to_calendar')
+      .eq('fk_journey_uuid', uuid);
+  return activities;
+}
+
+/**
+ * foreach activity fetch location and display marker in map
+ * @param activities all activities from a journey
+ */
+async function handleActivites(activities) {
```
Comment (Mathematti) [inline src/components/JourneyMap.vue]: spelling mistake

File: src/components/JourneyMap.vue
Comment (Mathematti) [inline src/components/JourneyMap.vue:1]: Is this fixable or is this an external issue?
grafik

File: src/components/JourneyMap.vue [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+        <p class="font-nunito-sans text-base">Die Karte basiert auf Eingaben bei der Reiseerstellung & neuen
+          Aktivitäten. Es kann daher zu falschen Darstellungen kommen.</p>
+      </div>
+    </div>
+  </div>
+</template>
+
+<style scoped>
+
+</style>
```
Comment (Mathematti) [inline src/components/JourneyMap.vue]: Remove empty style

File: src/components/JourneyMap.vue
Comment (EinSev) [reply src/components/JourneyMap.vue:1]: Shouldn't appear anymore

File: src/components/JourneyMap.vue (hunk 1 above)
Comment (EinSev) [reply src/components/JourneyMap.vue]: fixed

**Review by Mathematti — APPROVED** [review]

**Pull request merged**